scripts/release/release-test.R

```diff
@@ -36,1166 +36,136 @@ library(stringi)
-    cli_alert_success(paste0(table, " exists"))
-  } else {
-    exit_test(paste0(table, " doesn't exist"))
-  }
-  cli_alert_info(sprintf("Deleting [%s]", project2))
-  armadillo.delete_project(project2)
-} else {
-  cli_alert_warning("Testing basic authentication skipped, admin password not available")
-}
-
-cli_h3("Testing Rock profile")
-cli_alert_info(sprintf("Login to %s", armadillo_url))
-if(ADMIN_MODE) {
-  armadillo.login_basic(armadillo_url, "admin", token)
-} else {
-  armadillo.login(armadillo_url)
-}
-profile <- "rock"
-if (!as_docker_container) {
-  create_profile_if_not_available(profile, profiles$available, token, auth_type)
-}
-available_projects <- armadillo.list_projects()
-project3 <- generate_random_project_name(available_projects)
-available_projects <- c(available_projects, project3)
-cli_alert_info(sprintf("Creating project [%s]", project3))
-armadillo.create_project(project3)
-cli_alert_info(sprintf("Checking if project [%s] exists", project3))
-check_cohort_exists(project3)
-
-trimesterrep <- arrow::read_parquet(paste0(dest, "core/trimesterrep.parquet"))
-cli_alert_success("core/trimesterrep read")
-
-cli_alert_info("Uploading trimesterrep table")
-armadillo.upload_table(project3, "core", trimesterrep)
-rm(trimesterrep)
-cli_alert_success("Uploaded trimesterrep")
-
-cli_alert_info("Creating new builder")
-logindata <- create_dsi_builder(url = armadillo_url, profile = profile, password = admin_pwd, token = token, table = sprintf("%s/core/trimesterrep", project3))
-
-cli_alert_info(sprintf("Login with profile rock and table: [%s/core/trimesterrep]", project3))
-conns <- datashield.login(logins = logindata, symbol = "core_trimesterrep", variables = c("smk_t"), assign = TRUE)
-
-datashield.assign.table(conns, "core_trimesterrep", sprintf("%s/core/trimesterrep", project3))
-
-datashield.assign.expr(conns, "x", expr=quote(core_trimesterrep$smk_t))
-
-con <- create_ds_connection(password = admin_pwd, token = token, profile = profile, url = armadillo_url)
-if (con@name == "armadillo"){
-  cli_alert_success("Succesfully connected")
-} else {
-  cli_alert_danger("Connection failed")
-}
-dsDisconnect(con)
-
-ds_mean <- ds.mean("core_trimesterrep$smk_t", datasources = conns)$Mean
+source("test-cases/remove-data.R") # Add link_project once module works
+dm_clean_up(user = test_config$user, admin_pwd = test_config$admin_pwd, required_projects = list(project1), update_auto = test_config$update_auto, url = test_config$armadillo_url, skip_tests = test_config$skip_tests, interactive = test_config$interactive)
 datashield.logout(conns)
 
-cli_alert_info("Testing Rock profile mean values")
-verify_ds_obtained_mean(ds_mean, 61.059, 3000)
-
-armadillo.delete_table(project3, "core", "trimesterrep")
-armadillo.delete_project(project3)
-cat(sprintf("\nVerify in the UI that project [%s] and all its data is gone.", project3))
-wait_for_input()
-
+# NOT SURE WHAT THIS DOES OR ADDS TO PREVIOUS TESTS
```

Comment:
This test is for testing basic authentication whilst running in OIDC. So locally, it usually runs. Its value is debatable because on PR's we of course always run with basic auth, so its behaviour gets tested, but I like the idea of testing it locally as well.